README.md

```diff
@@ -1 +1,22 @@
-# OOP-School-Library
\ No newline at end of file
+# OOP-School-Library
+
+## 📑 Project description
+(v1.1)
+Create your own Enumerable
+
+## 🛠 Built with
+- Ruby
+
+## 📺 Live Demo
+- N/A
+
```

Comment:
- [OPTIONAL] To make your readme file more informative, it will be nice to add a `Getting started and Acknowledgements` sections as in this [tempate](https://github.com/microverseinc/readme-template).